exec/stdsql/querier.go

```diff
@@ -2,27 +2,49 @@ package stdsql
 
 import (
 	"context"
+	"errors"
 
 	"github.com/getsynq/dwhsupport/exec"
 	"github.com/getsynq/dwhsupport/exec/querier"
+	"github.com/getsynq/dwhsupport/sshtunnel"
 	"github.com/jmoiron/sqlx"
 )
 
 var _ querier.Querier[interface{}] = (*StdSqlQuerier[interface{}])(nil)
 
-func NewQuerier[T any](conn *sqlx.DB) *StdSqlQuerier[T] {
-	return &StdSqlQuerier[T]{conn: conn}
+type Querier_Opts[T any] func(querier *StdSqlQuerier[T])
+
+func Querier_WithSshTunnelDialer[T any](sshTunnelDialer *sshtunnel.SshTunnelDialer) Querier_Opts[T] {
+	return func(querier *StdSqlQuerier[T]) {
+		querier.sshTunnelDialer = sshTunnelDialer
+	}
+}
```

Comment:
possibility to add ssh tunnel dialer to stdsql logic which is used for sql based connections (postgres, mysql, redshift, ...)